### PR TITLE
chore(actors): finalize actor drivers & only alow calling actors local to module

### PR DESCRIPTION
**Pull request by NathanFlurry**

### CODE REVIEW

**Comment by NathanFlurry** [comment]
* **#418** <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb/418?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#417** <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb/417?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @NathanFlurry and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: deno.jsonc [hunk 1]
```diff
@@ -18,11 +18,10 @@
 		"docker:build": "docker build -t opengb -f cli.Dockerfile .",
 
 		// Generates schema
-		"artifacts:build:all": "deno task format && deno task artifacts:build:schema && deno task artifacts:build:runtime_archive && deno task artifacts:build:prisma_archive && deno task artifacts:build:dynamic_archive",
+		"artifacts:build:all": "deno task format && deno task artifacts:build:schema && deno task artifacts:build:runtime_archive && deno task artifacts:build:prisma_archive",
```
Comment (MasterPtato) [inline deno.jsonc:21]: The new generated sdks use the dynamic archive pretty heavily, how should we handle this future merge conflict?

File: deno.jsonc [hunk 2]
```diff
@@ -18,11 +18,10 @@
...
 		"artifacts:build:schema": "deno run --allow-env --allow-read --allow-write src/artifacts/build_schema.ts",
 		"artifacts:build:runtime_archive": "deno run --allow-env --allow-read --allow-write src/artifacts/build_runtime_archive.ts",
 		"artifacts:build:prisma_archive": "deno run --allow-env --allow-read --allow-write --allow-run src/artifacts/build_prisma_archive.ts",
-		"artifacts:build:dynamic_archive": "deno run --allow-env --allow-read --allow-write --allow-run src/artifacts/build_dynamic_archive.ts",
```
Comment (MasterPtato) [inline deno.jsonc:25]: See above comment

File: src/artifacts/build_dynamic_archive.ts [hunk 3]
```diff
@@ -1,13 +0,0 @@
...
-
-import { resolve } from "../deps.ts";
-import { buildArtifacts, rootSrcPath } from "./util.ts";
-
-await buildArtifacts({
-	rootPath: resolve(rootSrcPath(), "src", "dynamic"),
-	patterns: ["*.ts"],
-	outputPath: resolve(rootSrcPath(), "artifacts", "dynamic_archive.json"),
-	encode: "string",
-});
```
Comment (MasterPtato) [inline src/artifacts/build_dynamic_archive.ts:13]: <i class='graphite__hidden'>[Re: lines 1 to 14]<br/><br/></i>

See above comment<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb/417?utm_source=unchanged-line-comment">Graphite</a>.</span>

File: src/build/gen/module.ts [hunk 4]
```diff
@@ -164,31 +168,36 @@ function genActors(
...
+	// GeneratedCodeBuilder.wrap(
+	// 	"interface ActorsSnake {",
+	// 	actorsTypedefSnake,
+	// 	"}",
+	// );
+	// GeneratedCodeBuilder.wrap(
+	// 	"interface ActorsCamel {",
+	// 	actorsTypedefCamel,
+	// 	"}",
+	// );
```
Comment (MasterPtato) [inline src/build/gen/module.ts:196]: Dead code?

File: src/runtime/actor/drivers/cloudflare_durable_objects/driver.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
+// This file is only imported when the runtime is `cloudflare_workers_platform`.
+
+import { ActorDriver } from "../../driver.ts";
+import { Config } from "../../../mod.ts";
+import { __GlobalDurableObject } from "./global_durable_object.ts";
+
+// TODO: Fix Deno.env.get hack
+// TODO: doc performance requirements
+// TODO: names?
+// TODO: ctx.waitUntil
```
Comment (MasterPtato) [inline src/runtime/actor/drivers/cloudflare_durable_objects/driver.ts:10]: Should tickets be made for these?

**Review by MasterPtato — CHANGES_REQUESTED** [review]

**Pull request merged**